src/__tests__/jest/reducers/followReducer.test.js

```diff
@@ -0,0 +1,53 @@
+import expect from "expect";
+import followUnfollowReducer from "../../../redux/reducers/followUnfollowReducer";
+import {
+  FOLLOW_SUCCESS,
+  UNFOLLOW_SUCCESS,
+  FOLLOWING,
+  NOT_FOLLOWING
+} from "../../../redux/action_types";
+
+const initialState = {
+  follow: false,
+  message: {},
+  error: {}
+};
+
+const action = { message: {}, error: {}, follow: false };
+
+describe("Follow and unFollow reducer test", () => {
+  it("should return initial state when there is not action", () => {
+    expect(followUnfollowReducer(initialState.followUnfollow, action)).toEqual(
+      initialState
+    );
+  });
+  it("should handle when follow is default", () => {
+    expect(
+      followUnfollowReducer(initialState.followUnfollow, action).follow
+    ).toEqual(false);
+  });
+  it("should handle successful follow ", () => {
+    action.type = FOLLOW_SUCCESS;
+    expect(
+      followUnfollowReducer(initialState.followUnfollow, action).follow
+    ).toEqual(true);
```

Comment:
Empty lines. By now you should know how to properly structure your work to make your code cleaner.